src/screens/Customers/EditCustomerInformation.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Redirect, useHistory } from 'react-router-dom';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import { EMPTY_CUSTOMER } from '../../lib/redux/customerDataSlice';
+import { selectCurrentCustomer } from '../../lib/redux/customerData';
+import { selectAllCustomersArray } from '../../lib/redux/customerDataSlice';
+import TextField from '../../components/TextField';
+import Button from '../../components/Button';
+import { Switch, Typography } from '@material-ui/core';
+import { updateCustomer } from '../../lib/airtable/request';
+import { useFormik } from 'formik';
+import * as yup from 'yup';
+
+function EditCustomerInformation() {
+  const currentCustomer = useSelector(selectCurrentCustomer) || EMPTY_CUSTOMER;
+  const allCustomerNumbers = useSelector(selectAllCustomersArray).map(customer => customer.customerNumber);
+  const history = useHistory();
+  const [loading, setLoading] = useState(false);
+
+
+  const validationSchema = yup.object({
+    customerName: yup.string().required('Please enter a name'),
+    customerNumber: yup.number()
+      .min(0, 'Please enter a positive number')
+      .notOneOf(allCustomerNumbers, 'That customer number is used by another customer')
+      .required('Please enter a customer number'),
+    activeStatus: yup.bool(),
+  });
+
+  const formik = useFormik({
+    initialValues: {
+      customerName: currentCustomer.name,
+      customerNumber: currentCustomer.customerNumber.toString(),
+      activeStatus: currentCustomer.isactive,
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      handleSubmit(values)
+    }
+  })
+
+  if (currentCustomer === EMPTY_CUSTOMER) {
+    return <Redirect to='/customers' />
+  }
+
+  const handleSubmit = (values: any) => {
+    const { customerNumber, customerName, activeStatus } = values;
+    setLoading(true);
+
+    const customerUpdates = {
+      customerNumber: parseInt(customerNumber),
+      name: customerName,
+      isactive: activeStatus,
```

Comment:
Maybe inactivity should be taken off the meterType enum so that it can actually affect the look of the profile

Reply:
Good call. I removed it from the `meterType` enum (and Airtable) and replaced its uses in `CustomerProfile` with `customer.isactive` checks instead